index/scorch/segment/zap/build.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/Smerity/govarint"
 )
 
-const version uint32 = 6
+const version uint32 = 7
```

Comment:
minor thing, another earlier PR already have version 7, SO may be bump it once more? :)

Reply:
i agree and would go to 8

Reply:
cool, will do